daisy/workflow.go

```diff
@@ -543,6 +533,17 @@ func (w *Workflow) populate(ctx context.Context) DError {
 			return Errf("error populating step %q: %v", name, err)
 		}
 	}
+
```

Comment:
In IncludeWorkflow.populate(), there is also a call to validateVarsSubbed() and substituteSourceVars(), and they are still after the "populateStep" loop right now. I guess it should also be moved like what you did here.

Reply:
done!